int32 selectedUAV
bool isPositionHoldActiveFlag
bool isWPActiveFlag
bool isBuildingTrajectoryActiveFlag
trajectory_msgs/MultiDOFJointTrajectory multi_dof_trajectory_data

